components/Storyblok/SbStoryCard.tsx

```diff
@@ -85,7 +85,7 @@ export const SbStoryCard = ({
       imageFocus={cardFocus || storyCardFocus || heroFocus || bgFocus}
       tabColor={paletteAccentColors[value || storyTabColorValue]}
       link={link}
-      href={`/${full_slug}`}
+      href={full_slug}
```

Comment:
We remove all leading slash before we check for the prefix in the updated sbStripSlugUrl utility so we don't need to add a leading slash here anymore